.chloggen/olivierg_gateway-usage-gauge.yaml

```diff
@@ -1,16 +0,0 @@
-# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
why are we removing these?

Reply:
`chloggen update -v v0.25.0` removed these files automatically so this is expected.